.claude/commands/tests/test_exportcommands.py

```
@@ -781,6 +781,100 @@ def test_export_directory_uses_rsync_when_available(self):
+        path = os.path.join(self.claude_scripts, 'auth-cli.mjs')
+        self.assertTrue(
+            os.path.isfile(path),
+            f"Missing .claude/scripts/auth-cli.mjs — required by secondo-cli.sh and /secondo"
+        )
+
+    def test_secondo_cli_sh_present_in_claude_scripts(self):
+        """secondo-cli.sh must exist in .claude/scripts/ (source for export)."""
+        path = os.path.join(self.claude_scripts, 'secondo-cli.sh')
+        self.assertTrue(
+            os.path.isfile(path),
+            f"Missing .claude/scripts/secondo-cli.sh — required by /secondo command"
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)